### ISSUE TITLE
Feature Request: Add Vite option

### ISSUE THREAD

**Issue by riipandi** [msaaddev/new-tailwind-app#19]
Hi, thank you for creating this amazing tool. I have an idea. It will be nice if this tool has the option to use Vite.

Thanks.

**Comment by msaaddev** [comment]
Hey! I am actually using Vite under the hood to set up Vue3 project. Are you saying I should add support for other supported Vite frameworks?

**Comment by riipandi** [comment]
Yeah, I mean please add Vite to another project template too, such as React + Vite, etc.

**Comment by msaaddev** [comment]
That's a great idea. I am going to add this to the project board and will soon launch a release.
